### PR TITLE
Fix normalize_path on Windows

### PR DESCRIPTION
**Pull request by CharlesOkwuagwu**
This fixes `normalize_path ` on Windows.

### CODE REVIEW

File: apps/server/lib/mix/tasks/package.ex [hunk 1]
```diff
@@ -269,7 +269,6 @@ defmodule Mix.Tasks.Package do
       case priv_dir(app_name) do
         {:ok, priv_source_dir} ->
           File.cp_r!(priv_source_dir, priv_dest_dir)
-
```
Comment (zachallaun) [inline apps/server/lib/mix/tasks/package.ex:272]: Please keep this newline.

File: apps/server/lib/mix/tasks/package.ex [hunk 2]
```diff
@@ -323,11 +322,16 @@ defmodule Mix.Tasks.Package do
...
     case File.read_link(path) do
       {:ok, orig} ->
-        path
-        |> Path.dirname()
-        |> Path.join(orig)
-        |> Path.expand()
-        |> Path.absname()
+        case :os.type() do
+          {:win32, _} -> 
+            origin
```
Comment (zachallaun) [inline apps/server/lib/mix/tasks/package.ex:327]:
```suggestion
            orig
```

File: apps/server/lib/mix/tasks/package.ex [hunk 3]
```diff
@@ -323,11 +322,16 @@ defmodule Mix.Tasks.Package do
...
+           _ ->
```
Comment (zachallaun) [inline apps/server/lib/mix/tasks/package.ex:328]:
```suggestion

          _ ->
```

**Review by zachallaun — CHANGES_REQUESTED** [review]
Thank you for trying to address this!

There's a compilation error in this PR (`origin` doesn't exist). I'm also wondering if we can use CI to try to run `mix package` in a windows environment and make sure it succeeds.

File: apps/server/lib/mix/tasks/package.ex (hunk 1 above)
Comment (CharlesOkwuagwu) [reply apps/server/lib/mix/tasks/package.ex:272]: Cool. I guess we can close this then.

**Comment by scohen** [comment]
What was the bug? There's no description in the PR, and it looks to me like this won't handle relative paths on windows.

**Comment by scohen** [comment]
I'm closing this, as I don't think it will have any meaningful effect on whether or not lexical runs on windows.

**Pull request closed without merging**